mvrx-rxjava2/src/test/kotlin/com/airbnb/mvrx/ViewModelSubscriberTest.kt

```diff
@@ -778,14 +778,14 @@ class ViewModelSubscriberTest : BaseTest() {
     fun testCancelledIfOwnerDestroyed() {
         val job = viewModel.onEachInternal(owner) {}
         owner.lifecycle.handleLifecycleEvent(Lifecycle.Event.ON_DESTROY)
-        assertTrue(job.isCancelled)
+        assertTrue(job.isCompleted)
```

Comment:
About `isCancelled` -> `isCompleted` change: 
It caused by https://github.com/airbnb/MvRx/pull/444/files#diff-1a463601d931dd44cddf41594c27099aR74

Before, flow used to complete exceptionally because of `channel.cancel()`. It may lead to cancellation of collector's scope (if collector scope lives longer than viewModel/lifecycle).

@elihart @gpeal